acceptance/tests/pxp-module-puppet/negative/attempt_puppet_flag_whitelist_bypass.rb

```diff
@@ -6,6 +6,7 @@
 
   step 'pick test host' do
     @test_host = agents.first
+    skip_test 'Test not windows compatible' if @test_host.platform =~ /windows/
```

Comment:
@zreichert so this is still failing because line https://github.com/zreichert/pxp-agent/blob/eb91202bddd6da394bd5f80fff19dc8debe8608c/acceptance/tests/pxp-module-puppet/negative/attempt_puppet_flag_whitelist_bypass.rb#L5 causes `@agents` array to be empty when `windows` is the only agent platform, so then `[].first` is nil.